src/app/scripts/cac/pages/cac-pages-directions.js

```diff
@@ -23,6 +23,12 @@ CAC.Pages.Directions = (function ($, _, DirectionsList, Itinerary, Settings) {
     }
 
     Directions.prototype.initialize = function () {
+        
+        moment.updateLocale('en', {
+            relativeTime : {
+                mm: "%d min",
```

Comment:
Super tiny nitpick, but our code convention is to prefer single quotes for JavaScript strings.